fortuna/src/api/revelation.rs

```diff
@@ -154,3 +158,8 @@ impl Blob {
         }
     }
 }
+
+/// It expects base_uri to not end with /
+pub fn get_register_uri(base_uri: &str, chain_id: &str) -> String {
+    format!("{0}/v1/chains/{1}/revelations", base_uri, chain_id)
```

Comment:
I think we should register `{0}/v1/chains/{1}`, which allows us to add more endpoints underneath if needed.

Reply:
Also, I suggest using a crate to handle this path joining problem so you don't have to worry about the trailing slashes. Code snippet here from chat GPT demonstrating how to do it:

```
use url::Url;

fn main() {
    // Base URL
    let base_url = Url::parse("https://example.com").expect("Invalid base URL");

    // Path to be joined
    let path = "/some/path";

    // Join base URL with the path
    let result_url = base_url.join(path).expect("Failed to join URL and path");

    // Print the result URL
    println!("Result URL: {}", result_url);
}
```